### PR TITLE
Updating drawing examples with agent selection demo, closes #40

### LINKED ISSUE #40: Add the player selection command to examples

### PR DESCRIPTION
**Pull request by pmacalpine**
…nt annotation demo to c++ example (there was only an example of agent annotaion in the java example).

This is for issue https://github.com/Gama11/RoboViz/issues/40.  Examples have been tested.

### CODE REVIEW

File: examples/cpp/rvdraw.h [hunk 1]
```diff
@@ -186,7 +186,7 @@ unsigned char* newAnnotation(const string* text, const float* p,
 unsigned char* newAgentAnnotation(const string* text, bool leftTeam,
     int agentNum, const float* color, int* bufSize) {
 
-  *bufSize = (text == NULL) ? 3 : 7 + text->length;
+  *bufSize = (text == NULL) ? 3 : 7 + text->length();
```
Comment (Gama11) [inline examples/cpp/rvdraw.h]: Did this compile at all before?

**Comment by Gama11** [comment]
Very nice, thanks!

Btw, if you add "closes #issuenumber" to the title of the pull request (like I just did), the issue will be cloesd automatically when it's merged.

File: examples/cpp/rvdraw.h (hunk 1 above)
Comment (pmacalpine) [inline examples/cpp/rvdraw.h]: No, this line was not compiling in its original form.

Comment (Gama11) [inline examples/cpp/rvdraw.h]: I guess #51 is more important than I thought. :)

**Pull request merged**